components/ws-manager/pkg/manager/annotations.go

```diff
@@ -63,6 +63,9 @@ const (
 	// fullWorkspaceBackupAnnotation is set on workspaces which operate using a full workspace backup
 	fullWorkspaceBackupAnnotation = "gitpod/fullWorkspaceBackup"
 
+	// pvcWorkspaceFeatureAnnotation is set on workspaces which are using persistent_volume_claim feature
+	pvcWorkspaceFeatureAnnotation = "gitpod/pvcFeature"
```

Comment:
(newer annotations use the more "correct" domain-style naming)

```suggestion
	pvcWorkspaceFeatureAnnotation = "gitpod.io/pvcFeature"
```